assets/xml/objects/object_sa.xml

```diff
@@ -0,0 +1,101 @@
+        <Animation Name="gSariaHandsOutAnim" Offset="0x1186C"/>
+        <Animation Name="gSariaTransitionHandsSideToHips" Offset="0x11C38"/>
+        <Animation Name="gSariaHandsBehindBackWaitAnim" Offset="0x12100"/>
+        <Animation Name="gSariaTransitionHandsSideToChestToSide" Offset="0x1291C"/>
+        <Animation Name="gSariaRightArmExtendedWaitAnim" Offset="0x13008"/>
+        <Animation Name="gSariaGiveLinkOcarinaAnim" Offset="0x135EC"/>
+        <Animation Name="gSariaHoldOutOcarinaAnim" Offset="0x13CD8"/>
+        <Animation Name="gSariaHoldOcarinaAnim" Offset="0x140BC"/>
+        <Animation Name="gSariaStandHandsOnHipsAnim" Offset="0x14470"/>
+        <Animation Name="gSariaExtendRightArmAnim" Offset="0x14850"/>
+        <Animation Name="gSariaHandsOnFaceAnim" Offset="0x15220"/>
+
+        <DList Name="gSariaHeadDL" Offset="0x6BF8"/>
+        <DList Name="gSariaLeftHandDL" Offset="0x7900"/>
+        <DList Name="gSariaRightHandDL" Offset="0x7A40"/>
+        <DList Name="gSariaRightHandAndOcarinaDL" Offset="0x7B80"/>
+        <DList Name="gSariaChestDL" Offset="0x9918"/>
+        <DList Name="gSariaRightShoulderDL" Offset="0x9AA8"/>
+        <DList Name="gSariaRightArmDL" Offset="0x9C70"/>
+        <DList Name="gSariaLeftShoulderDL" Offset="0x9DA8"/>
+        <DList Name="gSariaLeftArmDL" Offset="0x9F70"/>
+        <DList Name="gSariaWaistDL" Offset="0xA0A8"/>
+        <DList Name="gSariaRightThighDL" Offset="0xA300"/>
+        <DList Name="gSariaRightShinDL" Offset="0xA550"/>
+        <DList Name="gSariaRightFootDL" Offset="0xA810"/>
+        <DList Name="gSariaLeftThighDL" Offset="0xA9D0"/>
+        <DList Name="gSariaLeftShinDL" Offset="0xAC20"/>
+        <DList Name="gSariaLeftFootDL" Offset="0xAEE0"/>
+
+
+        <Texture Name="gSariaClothesTULT" OutName="clothes_tult" Format="rgb5a1" Width="18" Height="6" Offset="0x21F0"/>
+        <Texture Name="gSariaMouthTULT" OutName="mouth_tult" Format="rgb5a1" Width="18" Height="6" Offset="0x2CF8"/>
+        <Texture Name="gSariaEyeTULT" OutName="eye_tult" Format="rgb5a1" Width="18" Height="6" Offset="0x2B00"/>
+        <Texture Name="gSariaSkinTULT" OutName="skin_tult" Format="rgb5a1" Width="8" Height="9" Offset="0x2A70"/>
+        <Texture Name="gSariaUnkTULT" OutName="unk_tult" Format="rgb5a1" Width="16" Height="9" Offset="0x2BD8"/>
+
+        <Texture Name="gSariaBootsTex" OutName="boots" Format="ci8" Width="16" Height="16" Offset="0x23F0"/>
+        <Texture Name="gSariaGreenTex" OutName="green" Format="ci8" Width="8" Height="8" Offset="0x24F0"/> 
+        <Texture Name="gSariaKneeTex" OutName="knee" Format="ci8" Width="16" Height="16" Offset="0x2570"/>
+        <Texture Name="gSariaTorsoTex" OutName="torso" Format="ci8" Width="16" Height="16" Offset="0x2670"/>
+        <Texture Name="gSariaBeltTex" OutName="belt" Format="ci8" Width="16" Height="16" Offset="0x2770"/>
+        <Texture Name="gSariaUnkGreenTex" OutName="IDK" Format="ci8" Width="16" Height="8" Offset="0x2870"/>
```

Comment:
This OutName doesn't seem appropriate.

Reply:
that wasn't supposed to be left in lol. Ill take it out in the next PR.